app/Domains/RabbitMQ/Jobs/MakeAccountJob.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Domains\RabbitMQ\Jobs;
+
+use App\Data\Models\Account;
+use Lucid\Units\Job;
+
+class MakeAccountJob extends Job
```

Comment:
great to have these jobs! However, there are two things that may improve it:

1. These jobs do not belong to `RabbitMQ` domain, because `Account` is not a `RabbitMQ` property. To know which domain to add a job to, ask yourself the question: "who's Account is it?". The answer here would be User/Talent (for example).
2. passing the entire `message` into the job. If i'm working on this code and i would like to reuse this Job to construct and `Account` instance, how would i know that the array should have `insights.account`? I would instead expect to pass the array right away instead of  in nested keys. This also has to do with the responsibility of the job, it shouldn't concern itself with where the message is coming from, but rather what it needs to return the object only.

These points also apply to the other factory jobs.